CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Change image lookup format for base OS image. `internal.image.variant` is set to "1" for this kubernetes version. This is a **breaking change** that requires manual steps. For the next kubernetes versions, `internal.image.variant` should not be set.
```

Comment:
Sorry. I didn't get this part. Do we expect users to set the variant? Since it is under `internal`, users shouldn't set anything for the image name or variant. They should just set k8s version and it should work. What is missing here?

Reply:
I know, it's a bit confusing. So, your right; users are not expected to set this or know about this, it's for internal use only.

The **breaking change** is when user sets newer/older Kubernetes version, it will not work. That's because, image with variant are manually built and only available for this Kubernetes version v1.25.16.

The use of variant is because `v1.25.16` is the last `v1.25` release, and we can't bump to newer `v1.25` release unless we move to `v1.26`.

Reply:
Wow. This doesn't seem healthy to me. We will not be able to override the k8s version after this PR. Right? I think we should find a better way.

Reply:
Yes, k8s version is pinned in this case. As you said, let's discuss this in KaaS sync to find a way around this.